app/views/applications/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<h1><%= @application.name %>'s Application</h1>
+<p>Address: <%= @application.full_address %></p>
```

Comment:
Could these application details be encapsulated into a partial for use in the admin show page as well?